packages/shared/src/domain/address/state.ts

```diff
@@ -1,17 +1,15 @@
+import { BadRequestError } from "../../error/bad-request";
+
 export function normalizeStateSafe(state: string): USState | undefined {
-  if (Object.values(states).includes(USState[state as keyof typeof USState])) {
-    return USState[state as keyof typeof USState];
-  } else if (states[state]) {
-    return states[state];
-  } else if (state === "DC") {
-    return USState.DC;
-  }
-  return undefined;
+  const keyFromEntries = Object.entries(states).find(
+    ([key, value]) => key === state || value.toLowerCase() === state.toLowerCase()
```

Comment:
My point is that California the *value* in the entry is `California` (which is title case). If we take `CALIFORNIA` and title case it, we get `California`. There's no need to make them both lower.

Reply:
I guess this would break if we introduce none-title case states (don't see how) but we can leave it. I agree it doesn't affect the outcome (just seems unnecessary).

Reply:
done (left w/ tolowecase)